libraries/Microsoft.Bot.Builder/DictionaryExtensions.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder
+{
+    /// <summary>
+    /// Extension method on object <see cref="object"/>.
+    /// </summary>
+    public static partial class DictionaryExtensions
+    {
+        private static JsonSerializer _coerceSerializer = new JsonSerializer() { TypeNameHandling = TypeNameHandling.None };
+
+        /// <summary>
+        /// Extension Method on object to coerce the value of the dictionary to type T.
```

Comment:
The documentation seems wrong, I think this is an extension method on IDictionary, not object.